metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraXmlRetrieverThroughDatabase.java

```diff
@@ -47,6 +51,26 @@ public String getXml(String accession) {
         }
     }
 
+    @Override
+    public Map<String, String> getXmls(List<String> accessionList) {
```

Comment:
this method is particularly used for gettingSampleXmls so it should be documented well by the code.

Reply:
Like setEnaObjectQuery, this method can also be implemented only in  SraXmlRetrieverThroughDatabase.

Reply:
ObjectImporter.java->importAnalysis()->importSamples()->sraXmlRetrieverByAccession.getXmls() will get called by API aswell

Reply:
you can call getXml for it